packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -363,9 +363,15 @@ export class DefaultWeb3Handler
           block['transactions'].map(async (transaction) => {
             transaction['hash'] = await this.getOvmTxHash(transaction['hash'])
             let ovmTx = await this.getTransactionByHash(transaction['hash'])
-            transaction['from'] = ovmTx.from
-            transaction['to'] = ovmTx.to
-            transaction['gasLimit'] = ovmTx.gasLimit.toNumber()
+            Object.keys(transaction).forEach((key) => {
+              if(ovmTx && ovmTx[key]) {
+                if(typeof ovmTx[key] === "object") {
+                  transaction[key] = ovmTx[key].toNumber()
```

Comment:
Are all objects guaranteed to be numbers? Might be safer to add `&& ovmTx[key].hasOwnProperty('toNumber')` to the `if`

Reply:
Ah I was looking for `isBigNumber` which isn't in ethers.js but `hasOwnProperty('toNumber')` will to the trick! 👍  -> [Check that transaction objects are BigNumbers](https://github.com/ethereum-optimism/optimism-monorepo/pull/97/commits/1b771bd8dd29579658725c03264083d28284a9bd)